dvc/repo/live.py

```diff
@@ -31,8 +32,9 @@ def create_summary(out):
     index_path = render(
         out.repo, renderers, metrics=metrics, path=html_path, refresh_seconds=5
     )
-
-    webbrowser_open(index_path)
+    auto_open = out.repo.config["plots"].get("auto_open", False)
+    if auto_open and int(os.environ.get(DVCLIVE_OPEN, "1")):
```

Comment:
`plots.auto_open` vs `DVCLIVE_OPEN` are not symmetrical though. WDYT of `DVC_PLOTS_OPEN`?

Reply:
Another thought: should we be more generic and implement `core.no_browser` and `DVC_NO_BROWSER` instead?

Reply:
I guess it's a thing to discuss in #6091 then here.

Reply:
The env var, in this case, was added for VSCode specifically, not really for end-user usage (however, it's not clear yet if they will need it or not). The asymmetry was intentional in order to allow VSCode to reactive DVCLIVE HTML even if user had `plots.auto_open` enabled. 

If going for the general approach, is there a reason to have both a `dvc config` field and an env var for the same purpose?

Reply:
We've decided on the VSCode end, at least for now, to respect the user's configuration for browser opening. Something like `DVC_NO_BROWSER` could be useful in the future for us and/or other integrations should we get user requests for a VSCode extension feature to disable browser opening outside the config.